pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools-scm"]
```

Comment:
Do you have to activate `setuptools_scm` beyond this line?

Reply:
It doesn't seem to need anything else: when I run `python -m build .` all the files end up in the sdist. I vaguely recall that the version handling needs activation, but I'm not using it.

Reply:
Makes sense.